Tensile/Common.py

```diff
@@ -1189,7 +1189,7 @@ def getArchitectureName(gfxName):
     {"StaggerUMapping":           [ 0 ] },    # recommend [0,1]
     {"MagicDivAlg":               [ 2 ] },
     {"GlobalSplitU":              [ 1 ] },
-    {"GlobalSplitUAlgorithm":     [ "MultipleBuffer" ] },
+    {"GlobalSplitUAlgorithm":     [ "SingleBuffer" ] },
     {"GlobalSplitUSummationAssignmentRoundRobin": [ True ] },
```

Comment:
How is this 

> default GSU algorithm will change to multi-buffer

?

Reply:
SingleBuffer means putting each GSU output to same one buffer, so we have to use atomic.
MultipleBuffer means putting each GSU output to different buffer, so we can use buffer_store.
In MultipleBuffer, we use another Kernel to summation result from different buffer.
This parameter only take effect on HPA BF16/FP16 type before.
We enhance to let each type to use this multipleBuffer.
Default behavior of other type is using atomic, we change this parameter back to single buffer.